index.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
       if (google.key) {
         params.push('key=' + encodeURIComponent(google.key));
       }
+      if (typeof google.libraries !== 'undefined' && google.libraries.length > 0) {
+        params.push('libraries=' + google.libraries.join());
```

Comment:
wow! Haha I never used it without parameter, and thought that it'd join with an empty string! Comma is a weird default from a developer point of view (or maybe just mine lol), sounds more like a literate user default lol